src/ngraph_capture_variables.h

```diff
@@ -22,7 +22,8 @@ namespace tensorflow {
 
 namespace ngraph_bridge {
 
-Status CaptureVariables(Graph* graph);
+Status CaptureVariables(Graph* graph,
+                        std::vector<string> skip_these_nodes = {});
```

Comment:
Remove the default value. Make the caller explicitly provide all the parameters.

Reply:
Done. Removed the default value from here as well as the MarkForClustering, in the process had to touch some test files as well which were calling these functions directly.